src/platform/stm32wl/STM32_LittleFS.h

```diff
@@ -76,17 +76,13 @@ class STM32_LittleFS
      * code. User should not call these directly
      *------------------------------------------------------------------*/
     lfs_t *_getFS(void) { return &_lfs; }
-    void _lockFS(void) { xSemaphoreTake(_mutex, portMAX_DELAY); }
-    void _unlockFS(void) { xSemaphoreGive(_mutex); }
+    void _lockFS(void) { /* no-op */ }
+    void _unlockFS(void) { /* no-op */ }
```

Comment:
before accepting this, can we make sure the Flash SPI is not shared with the Radio SPI? Indeed FreeRTOS was only included for the locking, not actual threading.